src/common/func_pictures.py

```diff
@@ -1,13 +1,16 @@
+import os
+
 import pygame
-from src.common.constants import *
+
+from src.common.config import ConfigPath
 
 
 def load_image(file):
     """
     Loads an image, prepares it for play
     """
 
-    file = os.path.join(DIR_PICTURES, file)
+    file = os.path.join(ConfigPath.folder("pictures"), file)
```

Comment:
file is a "python name", use of another variable name !?

Reply:
It was a keyword in 2.x versions